lib/src/front_end/ast_node_visitor.dart

```diff
@@ -415,7 +415,12 @@ class AstNodeVisitor extends ThrowingAstVisitor<void>
 
   @override
   void visitFieldDeclaration(FieldDeclaration node) {
-    throw UnimplementedError();
+    modifier(node.externalKeyword);
+    modifier(node.staticKeyword);
```

Comment:
Are we handling `const` modifiers? For static const fields

Reply:
Oh, good catch. It is working correctly (I think `const` uses the same `.keyword` token in `VariableDeclarationList`. Added tests.